Library/Homebrew/global.rb

```diff
@@ -63,6 +63,8 @@
 require "os/global"
 
 module Homebrew
+  include Kernel
```

Comment:
Won't be needing this anymore. Including this in the RBI file will do the work for us 😄